chatcommands.py

```diff
@@ -207,15 +207,16 @@ def check_blacklist(string_to_test, is_username, is_watchlist):
     reasons = list(set(question_reasons) | set(answer_reasons))
 
     # Filter out watchlist results
+    filter_out = ["potentially bad NS"]
```

Comment:
I doubt this would actually get filtered out because `x not in reason.lower()` would make the capitalized letters `NS` never hit, no?

Reply:
Heh, thanks :-) I'll try to fix tomorrow.